tests/test-dirs/parameters.t/run.t

```diff
@@ -281,6 +281,20 @@ we're correctly tracking parameters separately per file, even in server mode.
 
   $ query_errors reexport.mli
   [
+    {
+      "start": {
+        "line": 2,
+        "col": 18
+      },
+      "end": {
+        "line": 2,
+        "col": 19
+      },
+      "type": "warning",
+      "sub": [],
+      "valid": true,
+      "message": "Warning 49: no valid cmi file was found in path for module P. P is a parameter"
+    },
```

Comment:
@lukemaurer Please verify that this test change is appropriate

Reply:
Looks fine. Somewhat surprising that this comes up now but I don't think it hurts